Source/Core/InputCommon/ControlReference/ExpressionParser.cpp

```diff
@@ -945,6 +947,7 @@ static std::unique_ptr<Expression> ParseBarewordExpression(const std::string& st
   qualifier.control_name = str;
   qualifier.has_device = false;
 
+  // This will only work with the default device
```

Comment:
Care to elaborate? What happens with a non-default device? Does it explode, does it do _nothing_?

Reply:
I will make this clearer